package.json

```diff
@@ -23,7 +24,9 @@
     "babel-core": "^6.26.0",
     "babel-plugin-external-helpers": "^6.22.0",
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-plugin-transform-runtime": "^6.23.0",
     "babel-preset-env": "^1.6.1",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
was this what you needed to get the tests to work in the end? The yearly presets are deprecated... but no big deal if it's just for the tests.

Reply:
Yes. I need this. :(